.travis.yml

```diff
@@ -15,6 +15,7 @@ script:
   - npm run build:rollup
   - npm --prefix=server test
   - bazel test --noshow_progress --noshow_loading_progress --test_output=errors //src/wasm/... //src/tests/...
+  - bazel build //particles/Native/Wasm:all
```

Comment:
Do we want the `--noshow_xxx` args here similar to what we have for test above? I'm just guessing perhaps this cleans up the CI output. I'm not sure whether they're relevant for `build` but I feel like probably they are?

Reply:
We could do a few `noshow`s. I'm leaning against it, because I like having a record of compiler warnings. I've noticed, for instance, that there are things we can clean up on the kotlin side from our initial compilations.

Reply:
... eh. I just looked at the travis logs. On second thought, a lot of the progress indicators just clog the build log.